frontend/apps/filemanager/filemanager.lua

```diff
@@ -299,7 +299,7 @@ function FileManager:setupLayout()
                         file_manager:showOpenWithDialog(file)
                     end,
                 },
-                filemanagerutil.genBookInformationButton(file, self.book_props, close_dialog_callback),
+                filemanagerutil.genBookInformationButton(doc_settings_or_file or file, self.book_props, close_dialog_callback),
```

Comment:
That reads... weird. Is `doc_settings_or_file` not already doc_settings or file? ;).

Reply:
Thanks! Misprint fixed in https://github.com/koreader/koreader/pull/12324.